packages/common/src/services/PaymentService.ts

```diff
@@ -0,0 +1,93 @@
+import { inject, injectable } from 'inversify';
+
+import { API_ACCESS_BRIDGE_URL, INTEGRATION_TYPE } from '../modules/types';
+import type { IntegrationType } from '../../types/config';
+import type { Product } from '../../types/payment';
+import { logError } from '../logger';
+import { getNamedModule } from '../modules/container';
+
+import AccountService from './integrations/AccountService';
+
+@injectable()
+export default class PaymentService {
```

Comment:
Should this be renamed 🤔? This is actually the `AccessBridgeService` integration and not a payment service. It might be confusing with the CheckoutService.

Reply:
The Access related methods are in the AccessService, these are the payment related definitions that will be the one source of truth hopefully :D. It is confirmed with @dbudzins for the naming.